### DISCUSSION TITLE
unzip filz with Angular and zip.js

### DISCUSSION THREAD

**Discussion by tabi92** [gildas-lormeau/zip.js#308] [Q&A]
Hi everybody,

I need help please. 
I am using angular 13. And I need to know to extract zip file using zip.js in Angular.

**Comment by eafiontzi** [comment]
Hello, I finally got zip.js working with Angular 11, installing it as an NPM dependency and importing it using [this](https://github.com/gildas-lormeau/zip.js/issues/212#issuecomment-765859473) comment.
